build.gradle

```diff
@@ -54,7 +54,7 @@ configure(allprojects) {
 		springBootVersion = project.findProperty("springBootVersion") ?: "2.1.4.RELEASE"
 		springVersion = project.findProperty("springVersion") ?: "5.1.6.RELEASE"
 		reactorVersion = project.findProperty("reactorVersion") ?: "Californium-SR6"
-		openServiceBrokerVersion = "3.0.0.RELEASE"
+		openServiceBrokerVersion = "3.0.3.RELEASE"
```

Comment:
I just published 3.0.4.RELEASE. Mind bumping this?

Reply:
Just force pushed it. Thanks for the release @royclarkson!